### DISCUSSION TITLE
Contributing to libigl development?

### DISCUSSION THREAD

**Discussion by lukkio88** [libigl/libigl#2179] [General]
Hello, is contributing to libigl something people can somewhat do?

**Comment by alecjacobson** [comment]
Yes! That's very welcome

**Comment by lukkio88** [comment]
And what is the procedure? I read in the libigl web page you would like to replace bits and pieces of mosek. As I'd like to get a bit of experience implementing these solvers I think I could contribute there.

**Comment by alecjacobson** [comment]
That'd be really cool. If you already see something you'd like to do then
you can just submit a PR when you're finished.

If you're looking for ideas then this might be a good thread to solicit
them.
